cmd/signozschemamigrator/schema_migrator/traces_migrations.go

```diff
@@ -1,3 +1,360 @@
 package schemamigrator
 
 var TracesMigrations = []SchemaMigrationRecord{}
+
+// move them to TracesMigrations once it's ready to deploy
+var TracesMigrations_Staging = []SchemaMigrationRecord{
+	{
+		MigrationID: 1000,
+		UpItems: []Operation{
+			CreateTableOperation{
+				Database: "signoz_traces",
+				Table:    "signoz_index_v3",
+				Columns: []Column{
+					{Name: "ts_bucket_start", Type: ColumnTypeUInt64, Codec: "DoubleDelta, LZ4"},
+					{Name: "resource_fingerprint", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "timestamp", Type: DateTime64ColumnType{Precision: 9}, Codec: "DoubleDelta, LZ4"},
+					{Name: "id", Type: FixedStringColumnType{Length: 27}, Codec: "ZSTD"},
```

Comment:
Why do we need `id`?

Reply:
For list view of traces we want window based loading of the explorer.

Reply:
Why can't the same be done with simple limit and offset?

Reply:
Yeah got your point, let me try it out.